app/build.gradle

```diff
@@ -18,16 +19,23 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
         }
     }
-}
+    packagingOptions {
+        resources {
+            excludes += [
+                    'error_prone/Annotations.gwt.xml',
+                    'third_party/java_src/error_prone/project/annotations/Annotations.gwt.xml',
+                    'third_party/java_src/error_prone/project/annotations/Google_internal.gwt.xml',
+                    'jsr305_annotations/Jsr305_annotations.gwt.xml'
+            ]
+        }
+    }
 
-ext {
-    appCompatVersion = '27.1.1'
 }
 
 dependencies {
     implementation project(':library')
-    implementation "com.android.support:recyclerview-v7:$appCompatVersion"
-    implementation "com.android.support:leanback-v17:$appCompatVersion"
-    implementation "com.android.support:appcompat-v7:$appCompatVersion"
+    implementation "androidx.recyclerview:recyclerview:1.3.0"
+    implementation "androidx.leanback:leanback:1.0.0"
+    implementation "androidx.appcompat:appcompat:1.6.1"
     implementation 'com.google.android.exoplayer:exoplayer:r1.5.14'
```

Comment:
Hi, r1.5.14 is not available now. How can you got this dependency? Do you have .jar file of this version? Otherwise this PR will not work anymore.